server/src/main/java/org/opensearch/extensions/proto/ExtensionRequestOuterClass.java

```diff
@@ -0,0 +1,956 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: server/src/main/java/org/opensearch/extensions/proto/ExtensionRequest.proto
+
+package org.opensearch.extensions.proto;
+
+/**
+ - * ExtensionRequest class autogenerated from ExtensionRequest.proto
+ - */
+public final class ExtensionRequestOuterClass {
```

Comment:
Hm ... this class is auto generated, shouldn't we generate it during the build (like the typical code generation flows)? (https://github.com/google/protobuf-gradle-plugin)

Reply:
This is nice, thanks @reta let me try getting this integrated.